alws/crud/distribution.py

```diff
@@ -104,41 +106,66 @@ async def add_distributions_after_rebuild(
             await pulp_client.create_rpm_publication(repo)
 
 
+async def get_existing_packages(pulp_client: PulpClient,
+                                repository: models.Repository):
+    return await pulp_client.get_rpm_repository_packages(
+        repository.pulp_href,
+        include_fields=['pulp_href', 'artifact', 'sha256', 'location_href',
+                        'arch'])
```

Comment:
I'm think this func can be usefull for some tasks in future, can we turn it into method for `PulpClient` class?
This func looks kinda strange, you can make it look better, but it's up to you